libs/community/langchain_community/agent_toolkits/openapi/planner.py

```diff
@@ -346,6 +378,7 @@ def create_openapi_agent(
     verbose: bool = True,
     agent_executor_kwargs: Optional[Dict[str, Any]] = None,
     allow_dangerous_requests: bool = False,
+    allow_operations: Sequence[str] = ('get', 'post'),
```

Comment:
This looks good.

Could you introduce a type for this so users don't have to guess what the set of supported operations is?

```python
Operation = Literal['GET', 'POST', 'DELETE', 'PUT', 'PATCH']
```

```suggestion
    allow_operations: Sequence[Operation] = ('GET', 'POST'),
```

Reply:
@eyurtsev I've added Literals, so could you please explain where exactly I have to document them? Should I add it here in [docs](https://github.com/langchain-ai/langchain/blob/master/docs/docs/integrations/toolkits/openapi.ipynb) or anywhere else?

Reply:
@eyurtsev I saw your remarks below but what you can say about documentation? I guess you missed my previous message in this unresolved conversation?